SPECS/openssh.spec

```
@@ -657,6 +658,9 @@ getent passwd sshd >/dev/null || \
 %endif
 
 %changelog
+* Mon Feb 24 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 7.4p1-23.3.2
```

Like other changelog entries, you could add the current pam_ssh_agent_auth package version, but as it is actually generated:
```suggestion
* Mon Feb 24 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 7.4p1-23.3.2 + 0.10.3-2.23.3.2
```